lib/functions.zsh

```diff
@@ -28,7 +29,14 @@ function open_command() {
               ;;
   esac
 
-  nohup $open_cmd "$@" &>/dev/null
+  # Use nohup command
+  if [ ${TMUX:-NOT_USE_TMUX} = "NOT_USE_TMUX" ]; then
```

Comment:
That is simply not needed. See https://github.com/robbyrussell/oh-my-zsh/pull/4520#issuecomment-149446951. The current version works correctly.